docs/deploy/deploy-external-image.md

```diff
@@ -0,0 +1,58 @@
+
+# Deploy an External Image
+
+This section of the documentation provides guidance on how to ***deploy an external image*** within the pipeline.
+
+Deploying an external image directly allows you to bypass application and version configurations, offering flexibility for non-standardized services or third-party containers. The deployment process is straightforward and integrated into the UI.
+
+To summarize, your workflow for deploying an external image is:
+
+1. **Prepare the external image**: Ensure it's built and pushed to a container registry accessible by the platform.
+\
+2. **Access the `New deployment` dialog in the UI**: Select the option to deploy an external image from the menu.
+
+   ![Menu showing external image deployment option](../images/deploy/new-deployment-menu.png)
+
+3. **Provide the external image uri and configure settings**: In the dialog, specify the image and adjust settings like resources, replicas, and public access.
+
+   ![External Image Deployment settings dialog](../images/deploy/deployment-image-dialog.png)
+
+## External Image Reference
+
+The external image reference specifies the image to deploy. It must include:
+
+- The container registry (e.g., DockerHub, AWS ECR, GCR).
+- The image name.
+- The tag (e.g., `latest`, or a specific version like `1.2.3`).
+
+Example external image reference:
+
+```plaintext
+dockerhub.io/myorg/my-service:1.2.3
+```
+
+!!! tip
+
+    Ensure your external image is accessible from the platform. For private repositories, ensure the correct credentials or access policies are in place.
+
+## Deployment Settings for External Images
+
+The deployment settings for external images are similar to application deployments, with slight differences. Below is a description of the main features:
+
+| Dialog Item          | Description                                                                 |
+|----------------------|-----------------------------------------------------------------------------|
+| **External Image**   | The full external image reference, including registry, name, and tag.      |
+| **Environment Variables** | Any environment variables your container needs can be specified here.      |
+| **Deployment Settings**  | Configure whether the image runs as a service or a job. Also, set CPU, memory, and replicas for scaling. |
+| **Public Access**    | Configure whether the container is accessible externally via a public URL. |
+| **State Management** | If state is enabled, a `state` folder is created to persist data across restarts. See [state management docs](./state-management.md). |
+| **Deployment Name**  | The name of the deployment. You can change it to something descriptive.    |
+
```

Comment:
do we need a Yaml example?

Reply:
I am pointing to the yaml configuration here https://github.com/quixio/quix-docs/blob/ac6a84447f3f3fc5995363811e4dc53e58082a2b/docs/deploy/deploy-external-image.md?plain=1#L53